api/controllers/v1/massif/find.js

```diff
@@ -19,12 +19,12 @@ module.exports = async (req, res) => {
     }
 
     // Populate entrances
-    massif.entrances = await MassifService.getEntrances(massif.id);
-    await NameService.setNames(massif.entrances, 'entrance');
+    // massif.entrances = await MassifService.getEntrances(massif.id);
```

Comment:
Why commenting ? just remove if not needed anymore ;)

Reply:
We wanted to keep it commented since we don't treat the request to retrieve the entrances yet, to keep a vision on how it's done for now.